packages/agent/src/agent/http/index.ts

```diff
@@ -124,6 +125,10 @@ export interface HttpAgentOptions {
    * @default 3
    */
   retryTimes?: number;
+  /**
+   * The fetch implementation to use. Defaults to the global fetch function.
+   */
+  delayStrategy?: BackoffStrategy;
```

Comment:
These names could be consistent, either delayStrategy / DelayStrategy or backoffStrategy / BackoffStrategy
```suggestion
  backoffStrategy?: BackoffStrategy;
```

Also the comment ("fetch implementation") looks like it was copy/pasted from elsewhere.